### DISCUSSION TITLE
Add "Page Relations" / "Graph page relations"

### DISCUSSION THREAD

**Discussion by stavros-k** [outline/outline#5640] [Ideas]
Obsidian has this thing that visualizes the relations between docs, AFAIK it uses links in pages to create those
![image](https://github.com/outline/outline/assets/47820033/0037df05-8c04-42d9-9b05-4256f0f8aed8)

Other software's have similar with different names, WikiJS calls it "Relation Radial" (I couldn't find a picture of that, but its under Administration/Pages section)

---

Why I want this?

When you have a lot of people writing docs, you usually missing the "glue" between them. This graph allows you to see where the "linking" is missing and where you should focus!

I think this will benefit more the larger teams, but its cool/fun for small teams too!

**Comment by vinicius507** [comment]
I would love to see this in Outline. Great suggestion!
